Dockerfile.assisted-service

```diff
@@ -58,9 +60,8 @@ ARG WORK_DIR=/data
 
 RUN mkdir $WORK_DIR && chmod 775 $WORK_DIR
 
-# downstream this can be installed as an RPM
-ARG OC_URL=https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz
-RUN curl -s $OC_URL | tar -xzC /usr/local/bin/ oc
+#TODO: Use officail oc client once it has ICSP support https://bugzilla.redhat.com/show_bug.cgi?id=1823143
+COPY --from=oc-image /oc /usr/local/bin/
```

Comment:
if this is the path, nice!

Reply:
I created the image so.. yes this is the path